shortlist/solutions/C1.md

```diff
@@ -0,0 +1,16 @@
+## Proof
+
+We tile the rectangle as so:
+[Figure 1] (where the corners are black)
```

Comment:
It's dangerous to rely on a diagram to illustrate something that isn't formally written in the text of the proof. There should be something written that leaves no doubt as to how the diagram should be interpreted and serves as a jumping off point for the rest of the discussion.